espanol/canudos.md

```diff
@@ -162,4 +162,6 @@ Revelo que venia del futuro y que le daba dos opciones:
 
 Indico a sus ayudantes que comenzaran los preparativos para el mayor banquete de la historia del imperio.
 
-[La eleccion](eleccion/eleccion.md)
\ No newline at end of file
+[La eleccion](eleccion/eleccion.md)
+
+El Consejero y Beatito coincidieron en que en el banquete se servirian [arepas](https://www.google.com.au/search?q=arepas&espv=2&biw=1680&bih=984&tbm=isch&tbo=u&source=univ&sa=X&ved=0ahUKEwjQ_vChv7TPAhXLjpQKHXflClAQiR4IpAE&dpr=1) de entrada. Y cada invitado podria comer [tantas como quisiera](exageracion/exageracion.md).
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.